kod/object/active/holder/nomoveon.kod

```diff
@@ -513,5 +513,23 @@ messages:
       return iResult;
    }
 
+   TryDeflect(what = $,caster=$)
+   {
+      local oDeflect, lRadiusEnchantment;
+
+      foreach lRadiusEnchantment in plRadiusEnchantments
+      {
+         if Send(Nth(lRadiusEnchantment,1),@TryDeflect,
```

Comment:
I think this should just be sent to Deflect (if present) rather than every radius enchantment. Note how the original implementation checked if the object was enchanted with Deflect and if so, sent the message to the spell. Hold off on fixing this one though, I'm having a look now to see if we can implement a C call that will get the list element if present (specifically, if it is worth adding).